src/main/resources/org/apache/fluo/resources/java-checkstyle.xml

```diff
@@ -30,6 +28,8 @@
   </module>
 
   <module name="TreeWalker">
+    <module name="SuppressWithNearbyCommentFilter"/>
```

Comment:
Checkstyle has different rules for processing Java files. Some just do their own thing, others inherit from some parent behavior, allowing them to perform their checks inside a particular context. I don't know the details of this module, or exactly how the TreeWalker works, but at some point in the newer version, this rule which we had been using was changed so that it only works within the TreeWalker. It had to be moved in order to work with the newer version of Checkstyle.